modules/virt-example-node-placement-affinity-hyperconverged-cr.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-virtualization-components.adoc
+
+[id="virt-example-node-placement-affinity-hyperconverged-cr_{context}"]
+= Example: Node placement with affinity in the HyperConverged Cluster CR
+
+In this example, `affinity` is configured so that infrastructure resources are placed on nodes that are labeled with `example.io/example-infra-key = example-value` and workloads are placed on nodes labeled with `example.io/example-workloads-key = example-workloads-value`. Nodes that have more than eight CPUs are preferred for workloads, but if they are not available, pods are still scheduled.
+
+[source,yaml]
+----
+...
```

Comment:
I think it would be helpful to add the metadata section, so it is clear which object is mentioned here
(`HyperConverged`)